reference/artifacts-with-artifactsrewrite/in-kubernetes-v2/index.md

```diff
@@ -0,0 +1,202 @@
+---
+layout: single
+title:  "Artifacts In Kubernetes (Manifest Based)"
+sidebar:
+  nav: reference
+---
+
+{% include toc %}
+
+Artifacts play an important role in the Kubernetes provider v2. Everything from
```

Comment:
We usually call this the "Kubernetes V2 Provider" (even though having V2 last might make more sense).

Reply:
Fixed!